core/src/test/java/com/graphhopper/GraphHopperTest.java

```diff
@@ -2582,6 +2583,10 @@ void testLoadingWithAnotherSpeedFactorFails() {
             hopper.importOrLoad();
         }
         {
+            // todonow: what do we want here: setting the flag encoder string obviously has no effect, because we
+            //          load the encoding manager from disk instead. should there be an error? and how? basically this
+            //          is where we deviate from our current approach and we would probably have to adjust the GH api
+            //          to make this clear
             GraphHopper hopper = new GraphHopper()
                     .setFlagEncodersString("car|speed_factor=9")
                     .setProfiles(new Profile("car").setVehicle("car").setWeighting("fastest"))
```

Comment:
This is basically what changed in this PR: We used to get an error unless we used the same `graph.encoded_values` and `graph.flag_encoders` we used for the import. But now these settings are just ignored when loading the graph, just like `prepare.ch.threads` for example. Unfortunately this is not so obvious when looking at the config file or the GraphHopper class' API. For the config file it might help to use separate sections for 'import' and 'load' so we can distinguish those settings that only affect the import from those that still can be changed when loading a graph.
Regarding the GraphHopper class' API I am a bit less concerned as I think that class should be split into an 'Importer' and a 'Loader' at some point anyway.